source/postprocess/visualization/velocity_residual.cc

```diff
@@ -0,0 +1,169 @@
+/*
```

Comment:
`VelocityResidual` is a bit misleading as a name, because the postprocessor only computes the residual for the surface. I would suggest renaming it to `BoundaryVelocityResidual`, this would also open the door for future extensions to other boundaries.